README.md

```diff
@@ -26,7 +26,7 @@ $ iptb shell 4
 $ ipfs cat QmNqugRcYjwh9pEQUK7MLuxvLjxDNZL1DH8PJJgWtQXxuF
 hey!
 ```
-
+Available plugins now: Local IPFS node (plugin_name: localipfs), Docker IPFS node (plugin_name: dockeripfs)
```

Comment:
Would also be nice to be able to list available plugins via the CLI. Doesn't have to be in this PR though, just mentioning.